Automattic/Gravatar-SDK-Android
# QuickEditor: Fix weird glitch when opening the QE

Closes #291

### Description

Soo it's a tricky one. I can' reliably replicate this but I've found two places that don't look great: 
1. Opening the QE
2. Coming back to the QE from the custom tab

The issue exists only if the soft keyboard was visible before launching the QuickEditor. Here's how nr 1 looks like on Pixel 6 (I couldn't reproduce this on an emulator.): 

https://github.com/user-attachments/assets/349ee52d-031b-4fc7-a292-f921e8a337d1

Unfortunately, we can't fix this from the QE itself. The moment QE is launched is too late. This needs to be handled in the demo app or other third-party apps. Here's how it looks:


https://github.com/user-attachments/assets/909abce8-b53b-483c-a8d2-32f6b1290f48

When working on this I started playing with edge to edge to make sure this is not related and because this will be the default in Android 15 I think we should make sure we handle that. I enabled the edge-to-edge in the demo app and updated the QE insets a bit. 

### Testing Steps

1. Start the demo app
2. Tap on the text field to open the keyboard
3. Tap "Update avatar" 
4. Notice the bottom sheet slides from the bottom without any weird jumps
